### PR TITLE
[SDK] Make VIP visible in endpoint. Set protocol and visibility if VI…

### PR DESCRIPTION
**Pull request by mbdcos**
…P already exists

### CODE REVIEW

File: sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/ResourceUtils.java [hunk 1]
```diff
@@ -320,6 +320,7 @@ public static Resource getExpectedRanges(
                 .getLabelsBuilder()
                 .addLabels(getVIPLabel(vipName, vipPort));
 
+        builder.setVisibility(DiscoveryInfo.Visibility.CLUSTER);
```
Comment (nickbp) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/ResourceUtils.java]: this is overriding the visibility setting at L319

Comment (nickbp) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/ResourceUtils.java]: Maybe add a comment saying that this is meant to update visibility from prior service versions (it looks like that's what this is for?)

Comment (mbdcos) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/ResourceUtils.java]: NamedVIPEvaluationStage.java:L95 
Task visibility should be enough, this is to ensure discovery visibility is always CLUSTER.

Comment (mbdcos) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/ResourceUtils.java]: PR takes so much time to pass all tests. Will add the comment in the next one.

**Pull request merged**